Makefile

```diff
@@ -81,3 +81,6 @@ haskell:
 
 rye-clean:
 	rm -rf rye-src
+
+rust:
+	rustc src/rust/compress.rs -O --test --out-dir build/rust && ./build/rust/compress
```

Comment:
Does the `build/` need to exist prior to running this or will the `rustc` create one?